components/authz-service/storage/postgres/migration/sql/74_pre_force_upgrade.up.sql

```diff
@@ -8,4 +8,11 @@ DROP FUNCTION migration_trigger_func();
 
 DROP FUNCTION valid_migration_state_change(migration_state, migration_state);
 
-COMMIT;
\ No newline at end of file
+-- we have updated the ingest endpoints so that their top-level namespace is `ingest`
+-- this will update any custom or legacy policies using the old action name
+UPDATE iam_statements SET actions='{ingest:*}' WHERE actions='{infra:ingest:*}';
+
+-- update ingest role to have the correct action "ingest:*"
+UPDATE iam_roles SET actions='{"ingest:*", "compliance:profiles:get", "compliance:profiles:list"}' WHERE id='ingest';
+
+COMMIT;
```

Comment:
Changing this migration will mean people don't get the change in existing VMs. Since we haven't shipped this yet that's probably fine but worth noting.

Reply:
No customers have gotten this migration yet so everyone will get this change on force-upgrade.